test/k8sT/manifests/l4-policy-demo.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: "cilium.io/v2"
+kind: CiliumNetworkPolicy
+description: "L7 policy for allowing all traffic via proxy in demo DS"
+metadata:
+  name: "l7-policy-demo"
```

Comment:
Should this say `l4-policy-demo`? Also, the description should probably mention L4 instead of L7.

Reply:
Right, will fix